config/jobs-chromeos.yaml

```diff
@@ -64,6 +64,15 @@ _anchors:
         - collabora-chromeos-kernel
```

Comment:
Let's revert the configuration for non-codec Tast tests to what it was before https://github.com/kernelci/kernelci-pipeline/pull/897 was merged - so just mainline, and stable-rc.